dev/format_python_code.sh

```diff
@@ -23,6 +23,7 @@ for paths in \
     "tests/python/" \
     "cvat/apps/quality_control" \
     "cvat/apps/analytics_report" \
+    "cvat/apps/consensus" \
```

Comment:
Please also copy the pyproject.toml file from any other app ([e.g. this one](https://github.com/cvat-ai/cvat/blob/develop/cvat/apps/quality_control/pyproject.toml)) to format the code correctly.